package.json

```diff
@@ -13,7 +13,7 @@
     "type-check": "tsc",
     "storybook": "storybook dev -p 6006",
     "storybook-clean": "rm -rf ./node_modules/.cache/storybook",
-    "build-storybook": "storybook build"
+    "build-storybook": "NODE_OPTIONS=--max_old_space_size=8192 storybook build"
```

Comment:
This is the suggested workaround for memory errors reported when building storybook. We use the same solution in the origination app but given the size of the groups ui storybook library, the amount here is double what is there.

Reply:
soon enough we'll need high-performance computing centers to do local development  ;-)